converters/google/mappers.go

```diff
@@ -116,6 +116,24 @@ func mappers() map[string][]mapper {
 				merge:   converter.MergeProjectIamMember,
 			},
 		},
+		"google_storage_bucket_iam_policy": {
+			{
+				convert: converter.GetBucketIamPolicyCaiObject,
+				merge:   converter.MergeBucketIamPolicy,
```

Comment:
Doesn't look like this is available in the vendored code? Do we need to update conversion?

Reply:
It's rebased on master. This function is [here](https://github.com/GoogleCloudPlatform/terraform-validator/blob/master/vendor/github.com/GoogleCloudPlatform/terraform-google-conversion/google/storage_bucket_iam.go#L17)